package.json

```diff
@@ -17,11 +17,17 @@
     "got": "^6.3.0"
   },
   "scripts": {
-    "lint": "eslint index.js"
+    "lint": "eslint index.js",
+    "test": "jest --verbose --watch"
```

Comment:
Let's use `jest --coverage` instead since we'll be integrating code coverage in the future.

You can create a new command `test-watch` for `--watch`